test/StakeManager.test.ts

```diff
@@ -114,18 +115,28 @@ contract('StakeManager', function ([_, relayManager, anyRelayHub, owner, nonOwne
       )
     })
 
-    it('with owner set', async function () {
-      before(async function () {
+    context('with owner set', function () {
+      beforeEach(async function () {
```

Comment:
don't "beforeEach" something that is needed once. we already have too many "beforeEach" which slow us down.
just keep it "before"

Reply:
Here it did not work. There are not so many tests in this `context`, so will research later.